libs/core-ui/src/lib/components/InteractiveLegendUtils.test.ts

```diff
@@ -0,0 +1,13 @@
+import { interactiveLegendStyles } from "./InteractiveLegend.styles";
+import { getColorBoxClassName } from "./InteractiveLegendUtils";
+
+describe("InteractiveLegend", () => {
+  it("should get the correct colorBox className", () => {
+    const classes = interactiveLegendStyles();
+    expect(getColorBoxClassName(0)).toEqual(classes.circleColorBox);

```

Comment:
nit: might be cleaner to create array of shapes and loop over check, something like this (coded without any checking):
```
shapes = [classes.circleColorBox, classes.squareColorBox, classes.diamondColorBox, classes.triangleColorBox, classes.triangleDownColorBox]
shapes.forEach((shape, idx) => {
    expect(getColorBoxClassName(idx)).toEqual(shape)
})
```

Reply:
updated, thanks!